app/src/parser_txdef.h

```diff
@@ -116,9 +116,9 @@ typedef struct {
   uint64_t total;
   uint64_t decimals;
   uint8_t default_frozen;
-  char unitname[8];
-  char assetname[32];
-  char url[32];
+  char unitname[9];
+  char assetname[33];
+  char url[33];
```

Comment:
URLs for assets in Algorand are now 96 bytes.  Shouldn't this be 97?
https://developer.algorand.org/docs/get-details/parameter_tables/#asa-constraints